src/screens/codeVerification/CodeVerification.tsx

```diff
@@ -1,181 +1,227 @@
                         delete user.mode
-                        signup(user).then(response=>{
+                        signup(user).then(response => {
                             refRBSheet.current.open()
                         })
-                        .catch(error=>{
-                            console.log(error.response); 
-                        })   
+                            .catch(error => {
+                                console.log(error.response);
+                            })
+                    }
+                })
+                .catch(error => {
+                    setIsLoading(false)
+                    console.log(error.response);
+                })
+        }
+    }
+
+    const verifyForgotPasswordHandler = () => {
+        console.log(route.params)
+        if (mode === 1) {
+            const { email } = route.params
+            const data = {
+                email: email,
+                otp: code
+            }
+            forgotPasswordVerifyEmailOtp(data).then(
+                response => {
+                    if(response?.status === 400){
+                        Alert.alert(response.data?.message)
+                        return
+                    } 
+                        console.log('res', response, 'data', data)
+                        setIsLoading(false)
+                        setCode("")
+                        navigation.navigate('SetNewPasswordScreen', response)
+                })
+                .catch(error => {
+                    console.log(error.response);
+                });
+        }
+        else {
+            const { country_code, phone_number } = route.params
+            const data = {
+                country_code: country_code,
+                phone_number: phone_number,
+                otp: code
+            }
+            forgotPasswordVerifyPhonelOtp(data).then(
+                response => {
+                    if (response?.status === 'success') {
+                        console.log('res', response, 'data', data)
+                        setIsLoading(false)
+                        setCode("")
+                        navigation.navigate('SetNewPasswordScreen', response)
                     }
-            })
-            .catch(error=>{
-                setIsLoading(false)
-                console.log(error.response); 
-            })
+                }
+            ).catch(error => {
+                console.log(error.response);
+            });
         }
     }
-    
-    return(
-        <View style={{marginTop : -10}}>
+
+    return (
+        <View style={{ marginTop: -10 }}>
             <ImageBackground source={Images.Background} resizeMode="cover" style={styles.image}>
-                <Header onPress={()=>navigation.goBack()} isBackIconVisible={true}/>
-                
-                <KeyboardAwareScrollView 
+                <Header onPress={() => navigation.goBack()} isBackIconVisible={true} />
```

Comment:
@simranahujacb Please set reference to method here instead of actually declaring a method and create a method outside return statement